setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
```

Comment:
Why do we have both a setup.py and a pyproject.toml?  Doesn't the former duplicate the latter?

Reply:
I would expect that we could publish with poetry

Reply:
Also, as a kinda neat sidenote: It looks like even pip now supports installing a package from a pyproject.toml file!  https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support

Reply:
see comment below